examples/telnet/as400_power_actions.js

```diff
@@ -129,16 +75,14 @@ function validate() {
 */
 function get_status() {
     getAS400Info()
-        .then(powerParsing)
-        .then(parseInfo)
         .then(D.success)
```

Comment:
since you have this in the getAS400Info

d.resolve(out);

It means you will pass the out to the D.success method which expects variables or nothing

Please refine the getAS400Info method to something that does not return values and rejects if the command fails